saml/lang/C/messages.po

```diff
@@ -0,0 +1,98 @@
+# ADDON saml
+# Copyright (C) 
+# This file is distributed under the same license as the Friendica saml addon package.
+# 
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-05-18 07:23+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
```

Comment:
Seems like the replacing mechanism didn't work

Reply:
Why? The source file (`lang/C/messages.po`) never had a revision. It will be added when the translated file are exported from Transifex.

Reply:
A I see